### ISSUE TITLE
Swift on Linux doesn't work with default settings

### ISSUE THREAD

**Issue by Cyberbeni** [codecov/codecov-bash#235]
This is my current codecov step during testing on Linux:
```
llvm-cov export -format="lcov" .build/debug/TypedNotificationCenterPackageTests.xctest -instr-profile .build/debug/codecov/default.profdata > info.lcov
bash <(curl -s https://codecov.io/bash) -J 'TypedNotificationCenter' -n 'Linux' -F 'Linux'
```
I think the first line should be covered by the default options.
Link to full workflow: https://github.com/Cyberbeni/TypedNotificationCenter/blob/master/.github/workflows/run_tests.yml

**Comment by drazisil** [comment]
HI @Cyberbeni 

To clarify, you are asking the bash uploader to convert the profdata to lcov for you?

**Comment by Cyberbeni** [comment]
Yes, just like how the script already handles the default settings output of running Swift tests via Xcode on macOS. https://github.com/codecov/codecov-bash/blob/cdd0d95b045e5dfcaa2aba9672314ade5ee941ff/codecov#L985
